eng/pipelines/libraries/base-job.yml

```diff
@@ -112,7 +112,7 @@ jobs:
       - ${{ if ne(parameters.liveCoreClrBuildConfig, '') }}:
         - template: /eng/pipelines/common/download-artifact-step.yml
           parameters:
-            unpackFolder: $(_coreClrDownloadPath)
+            unpackFolder: ${_CORECLRDOWNLOADPATH}
```

Comment:
Does this work when passed as a task arg? `${...}` is a shell scripting syntax, not known by cmd or AzDO as far as I know.

https://github.com/dotnet/runtime/blob/f991fe7d996b66d9f471fba23925947caeb7e6fe/eng/pipelines/common/download-artifact-step.yml#L18-L23

Reply:
Looks like not, macOS is choking:

> `/usr/bin/tar -xvf /Users/runner/runners/2.164.3/work/1/s/__download__/CoreCLRProduct_OSX_x64_release/CoreCLRProduct_OSX_x64_release.tar.gz -C /Users/runner/runners/2.164.3/work/1/s/${_CORECLRDOWNLOADPATH}`
> https://dev.azure.com/dnceng/public/_build/results?buildId=491593&view=logs&j=165846b8-a05f-59ec-06e6-141fc8c9f9e4&t=f25ddc43-e446-5261-e436-e8dc261f76c0&l=13

Then fails during the build trying to use `/Users/runner/runners/2.164.3/work/1/s/artifacts/transport/coreclr`.

Reply:
I think a workaround might be to run a script step on the affected Linux platforms that uses an AzDO command to reset the value of `_coreClrDownloadPath` to `${_CORECLRDOWNLOADPATH}`, then maybe we can continue to use `$(_coreClrDownloadPath)` in subsequent steps as normal.

Reply:
Or just use this directly in the declaration of the variable, instead of here... also we could condition that to use this syntax if container != ''

Reply:
For some reason I thought it was a yaml syntax thing. I just pushed a commit that only uses this in the `_coreClrArtifactsPathArg` declaration and when `container != ''`